paddle/fluid/pir/dialect/op_generator/ops_api_gen.py

```diff
@@ -120,6 +120,7 @@
 
 NO_NEED_GEN_STATIC_ONLY_APIS = [
     'add_n_',
+    'all_reduce',
```

Comment:
```suggestion
    'all_reduce',
    ‘all_reduce_’,
```
这个op有`inplace`属性，会自动生成`all_reduce_`，所以需要把`all_reduce_`加入

Reply:
已修改，多谢~